core/src/smartcontracts/isi/permissions/checks.rs

```diff
@@ -159,117 +163,131 @@ pub fn check_query_in_expression(
 ///
 /// # Errors
 /// If a user is not allowed to execute one of the inner queries,
-/// given the current `validator`.
+/// given the current `judge`.
```

Comment:
```suggestion
/// given the current [`Judge`].
```

Reply:
We brought this up on retro, having as many hyperlinks as possible is a good idea.